### DISCUSSION TITLE
What is the future that lxqt should take?

### DISCUSSION THREAD

**Discussion by telmobss1** [lxqt/lxqt#1957] [General]
In the other topic we were talking about how the lxqt project should accept donations from users who wanted to donate voluntarily 

But some ideas have emerged that could improve the project and this topic serves to discuss them 

One of the options is to create a non-profit foundation, but for that it takes a lot of bureaucracy

Another option is to create a company by partnering with other companies including hardware companies

Remembering that the topic is the continuation of the previous topic with the intention that the money given to the lxqt project is used in the best possible way 

Other ideas are welcome

**Comment by yan12125** [comment]
As a reference, previous discussions are in https://github.com/lxqt/lxqt/issues/1846#issuecomment-799576568

**Comment by wxl** [comment]
If paperwork, organizational structure, and tax burdens are what you refer to when you mean "bureaucracy," there's nothing that separates a company from a non-profit. In fact, it's arguably quite worse to be a company since, by definition, the tax liability is higher. There are plenty of open source projects out there that have foundations set up (look at KDE if you want a really good example) and they work just fine.

Companies require wanting to do *business*. Is that what we really want to do? I mean, sure, it would be nice to make money, but it also means having to create it and that involves a heck of a lot more than just committing code. It means sales and marketing, even if that is to hardware vendors. 

BTW if hardware vendors (ahem, Broadcom) don't even bother to work with the Linux kernel, why would they bother with a relatively small player in the Linux desktop environment world?

**Reply by telmobss1** [reply]
> Se papelada, estrutura organizacional e carga tributária são o que você chama de "burocracia", não há nada que separe uma empresa de uma organização sem fins lucrativos. Na verdade, é sem dúvida muito pior ser uma empresa, uma vez que, por definição, o passivo fiscal é maior. Existem muitos projetos de código aberto por aí que têm fundamentos configurados (veja o KDE se você quiser um exemplo realmente bom) e eles funcionam muito bem.
> 
> As empresas precisam querer fazer _negócios_ . É isso que realmente queremos fazer? Quero dizer, claro, seria bom ganhar dinheiro, mas também significa ter que criá-lo e isso envolve muito mais do que apenas comprometer código. Significa vendas e marketing, mesmo que seja para fornecedores de hardware.
> 
> A propósito, se os fornecedores de hardware (ahem, Broadcom) nem mesmo se importam em trabalhar com o kernel do Linux, por que eles se incomodariam com um player relativamente pequeno no mundo do ambiente de desktop Linux?

It really makes sense to think that way, so what we have to see is the best way to manage money
